### ISSUE TITLE
New update does not respect opacity setting for padding set with [window_padding_width 10]

### ISSUE THREAD

**Issue by av1155** [kovidgoyal/kitty#7895]
It's always been transparent, but since the last update [v0.36.3](https://github.com/kovidgoyal/kitty/releases/tag/v0.36.3), it launched with these solid borders and i cant get rid of them. I did not change anything.

## kitty.conf:
```bash
include dynamic.conf
include theme.conf

# font_size 9.5  #Handled by dynamic.conf
font_family      JetBrainsMono Nerd Font Mono
bold_font        auto
italic_font      auto
bold_italic_font auto

window_padding_width 10
# resize_in_steps yes
# background_opacity 0.7 #Handled by dynamic.conf
background_blur 64
hide_window_decorations yes
confirm_os_window_close 0

map alt+1 previous_tab
map alt+2 next_tab

tab_bar_edge                bottom
tab_bar_style               powerline
tab_powerline_style         slanted
tab_title_template          " {index} |{fmt.fg.red}{bell_symbol}{activity_symbol}{fmt.fg.tab} {title} "

underline_hyperlinks always
show_hyperlink_targets yes
```
## dynamic.conf (per-device)
```bash
font_size 17
background_opacity 0.7
macos_option_as_alt left
```

<img width="866" alt="image" src="https://github.com/user-attachments/assets/061fdca2-764b-4953-b613-8ac64a16ef7d">

## theme.conf
```bash
# vim:ft=kitty

## name:     Catppuccin Kitty Mocha
## author:   Catppuccin Org
## license:  MIT
## upstream: https://github.com/catppuccin/kitty/blob/main/themes/mocha.conf
## blurb:    Soothing pastel theme for the high-spirited!



# The basic colors
foreground              #cdd6f4
background              #1e1e2e
selection_foreground    #1e1e2e
selection_background    #f5e0dc

# Cursor colors
cursor                  #f5e0dc
cursor_text_color       #1e1e2e

# URL underline color when hovering with mouse
url_color               #f5e0dc

# Kitty window border colors
active_border_color     #b4befe
inactive_border_color   #6c7086
bell_border_color       #f9e2af

# OS Window titlebar colors
wayland_titlebar_color system
macos_titlebar_color system

# Tab bar colors
active_tab_foreground   #11111b
active_tab_background   #cba6f7
inactive_tab_foreground #cdd6f4
inactive_tab_background #181825
tab_bar_background      #11111b

# Colors for marks (marked text in the terminal)
mark1_foreground #1e1e2e
mark1_background #b4befe
mark2_foreground #1e1e2e
mark2_background #cba6f7
mark3_foreground #1e1e2e
mark3_background #74c7ec

# The 16 terminal colors

# black
color0 #45475a
color8 #A8A8A8

# red
color1 #f38ba8
color9 #f38ba8

# green
color2  #a6e3a1
color10 #a6e3a1

# yellow
color3  #f9e2af
color11 #f9e2af

# blue
color4  #89b4fa
color12 #89b4fa

# magenta
color5  #f5c2e7
color13 #f5c2e7

# cyan
color6  #94e2d5
color14 #94e2d5

# white
color7  #bac2de
color15 #a6adc8
```

**Comment by Error6929** [comment]
Yup, I am having the same issue. I spent my last hour rewriting my whole config... turns out it was indeed a bug in the terminal itself. I can't share the SS as I don't have setup my SS utilities, but you are not alone!

**Comment by oversys** [comment]
Came to open an issue and found this opened 39 minutes ago. Have the same problem, upgraded from 0.35.2 to 0.36.3 and these solid bars appeared. Using Arch Linux.

kitty 0.35.2
![image](https://github.com/user-attachments/assets/e965add9-7638-470b-9ebb-0db35ad7fe0e)

kitty 0.36.3
![image](https://github.com/user-attachments/assets/6f79db74-8bdd-4074-96d4-e95a1157b949)

**Comment by jude7733** [comment]
Yeah, same here on arch. I think there is no need of sharing a screenshot, right?. hoping it gets fixed soon.

**Comment by tonymccallie** [comment]
So will this be fixed in the next version? I'm seeing the same thing in 0.36.3 in Arch as well. Just updated

**Comment by alexbet** [comment]
Same here.

**Comment by fisher4men** [comment]
Have the same problem on gentoo

**Comment by kovidgoyal** [comment]
The fix for this was made 3 hours after it was reported and released a couple of days ago as kitty 0.36.4.  So please dont spam this issue.
